src/plugins/jma/datasource.py

```
@@ -0,0 +1,364 @@
+                                }
+                            )
+                            measurements.append(measurement)
+                            measurements_count += 1
+                            
+                            if limit and measurements_count >= limit:
+                                break
+                                
+                    if measurements:
+                        yield measurements
+                        
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)